libs/blocks/featured-article/featured-article.js

```diff
@@ -5,7 +5,9 @@ async function createCategoryLink(el, category = 'News') {
   Promise.all(promises).then(async ([taxonomyMod, helpersMod]) => {
     const fetchTaxonomy = taxonomyMod.default;
     const { updateLinkWithLangRoot } = helpersMod;
-    const taxonomy = await fetchTaxonomy(getConfig(), '/topics');
+    const config = getConfig();
```

Comment:
`getConfig` doesn't need to be called for each promises. It could be moved before `Promise.all`.

Or before the block containing the line: https://github.com/adobecom/milo/blob/3e2a0b19d26cc5234c158d451699b2d4a919821b/libs/blocks/recommended-articles/recommended-articles.js#L79

Reply:
The code after Promise.all only executes once. Promise.all turns the two promises into one and only resolves when both of them are complete.

I'll take a look at the recommended articles, though, and see about optimization there

Reply:
good point :)